lib/model/query/users.js

```diff
@@ -86,10 +87,13 @@ const emailEverExisted = (email) => ({ maybeOne }) =>
   maybeOne(sql`select true from users where email=${email} limit 1`)
     .then((user) => user.isDefined());
 
+const getPreferences = (userId) => ({ one }) => getForUser(userId)({ one });
```

Comment:
This line is doing weird things with containers... (the `{ one }` argument is destructuring the container)

See my other comment about just using `getForUser` from the UserPreferences query module directly.